### ISSUE TITLE
Tab Accessibility Support

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#55]
Tab Accessibility Support

> Describe in detail the issue you're having.
  Looking at the tab widget to help meet our accessibility requirement. I see the left/ right arrows partially work. Per the guideline (https://www.w3.org/TR/wai-aria-practices/#tabpanel):
    Left Arrow: moves focus to the previous tab. **If focus is on the first tab, moves focus to the last tab.** Optionally, activates the newly focused tab (See note below).
    Right Arrow: Moves focus to the next tab. **If focus is on the last tab element, moves focus to the first tab.** Optionally, activates the newly focused tab (See note below).

I've played with the widget in the storybook, I'm not seeing the arrows do the additional navigation for going to the first or last tab (in bold above). 

Also I noticed the space bar does not activate the tab either.  Per guideline:
 Space or Enter: Activates the tab if it was not activated automatically on focus.

> Is this a feature request (new component, new icon), a bug, or a general issue?
accessibility enhancement

    Is this issue related to a specific component?
    Tabs

    What offering/product do you work on? Any pressing ship or release dates we should be aware of?
    IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

**Comment by alwilkin** [comment]
Can you provide an ETA for delivery?

**Comment by johnharan** [comment]
Looks like the space key still does not activate a tab, only enter. 

However, looking at the latest carbon tabs implementation (http://www.carbondesignsystem.com/components/tabs/code), it seems that the the left/right arrows now both change focus and activate the focused tab.
